File: core/src/main/java/net/minestom/server/utils/math/ShortRange.java
package net.minestom.server.utils.math;

public class ShortRange extends Range<Short> {

  public ShortRange(Short minimum, Short maximum) {
    super(minimum, maximum);
  }

  public ShortRange(Short value) {
    super(value);
  }

  /** {@inheritDoc} */
  @Override
  public boolean isInRange(Short value) {
    return value >= this.getMinimum() && value <= this.getMaximum();
  }
}

File: src/main/java/net/minestom/server/utils/math/ShortRange.kt
package net.minestom.server.utils.math

class ShortRange : Range<Short> {
    constructor(minimum: Short, maximum: Short) : super(minimum, maximum) {}
    constructor(value: Short) : super(value) {}

    /** {@inheritDoc}  */
    override fun isInRange(value: Short): Boolean {
        return value >= minimum && value <= maximum
    }
}